images/macos/provision/utils/utils.sh

```diff
@@ -96,4 +96,15 @@ get_default_xcode_from_toolset() {
 verlte() {
     sortedVersion=$(echo -e "$1\n$2" | sort -V | head -n1)
     [  "$1" = "$sortedVersion" ]
+}
+
+brew_install_ignoring_sha256() {
+    local TOOL_NAME=$1
+
+    CASK_DIR=/usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask/Casks
```

Comment:
Let's avoid hardcoding this path. Use `brew --repo homebrew/cask`

Reply:
Fixed it